src/SCRIPTS/TOOLS/bf.lua

```diff
@@ -2,6 +2,7 @@ local toolName = "TNS|Betaflight setup|TNE"
 chdir("/SCRIPTS/BF")
 
 apiVersion = 0
+craftName = nil
```

Comment:
We would have to change this to ```mcuId``` and also update all the other variables/filenames etc that contains ```craftName```.